src/course-search/FacetListBase.jsx

```diff
@@ -65,16 +68,28 @@ const FacetListBase = ({
   );
 
   return (
-    <FacetDropdown
-      items={renderItems()}
-      title={title}
-      isBold={isBold}
-    />
+    typeahead ? (
```

Comment:
tiny nit/suggestion: Rather than using a ternary conditional here, it might be slightly more readable as 2 separate return statements?

```
if (typeahead) {
  return <TypeaheadFacetDropdown />;
}

return <FacetDropdown />
```